atc/atccmd/command.go

```diff
@@ -958,7 +957,7 @@ func (cmd *RunCommand) constructBackendMembers(
 				&scheduler.Scheduler{
 					Algorithm: alg,
 					BuildStarter: scheduler.NewBuildStarter(
-						factory.NewBuildFactory(
+						builds.NewPlanner(
```

Comment:
Really minor, and I don't really have strong opinions on this either way, but is there a reason you went with `builds` instead of `build`? I feel like Go packages tend to be singular (with the exception of `bytes`, `strings`, `errors`, etc., but if I had to guess, I'd say they mainly used the plural so as not to conflict with the built-in types of the same name). Plus, `build.Planner` feels just slightly easier to say than `builds.Planner` to me

Reply:
@aoldershaw Following the same pattern as those packages you mentioned. The pluralization also avoids clashing with local vars; `var build build.Build` actually shadows the `build` package from then on so it'd be a bit problematic. I suppose the same is true for `var builds []build.Build`, though I imagine that'd be less common.

I think the plural form also makes the package feel more like a "domain" than worrying about whether the contained types really deal with a single build or deal with the general concept of builds. That's all kind of subtle semantics though. 🤷‍♂️